src/sql/magic.py

```diff
@@ -125,6 +126,7 @@ def __init__(self, shell):
         # Add ourself to the list of module configurable via %config
         self.shell.configurables.append(self)
 
+    @no_var_expand
```

Comment:
this looks like a merge error? the `@no_var_expand` should go in the execute method?

Reply:
You are right. Somehow it's moved.

Fixed